codebases/omgevingsbeleid.html

```diff
@@ -106,7 +106,7 @@ <h3>Find out more</h3>
       <li><a href="https://github.com/Provincie-Zuid-Holland/Omgevingsbeleid-Frontend" target="_blank">Github repository (frontend)</a></li>
       <li><a href="https://github.com/Provincie-Zuid-Holland/Omgevingsbeleid-API" target="_blank">Github repository (backend)</a></li>
       <li><a href="https://omgevingsbeleid.zuid-holland.nl/" target="_blank">Demo website</a></li>
-      <li><a href="https://github.com/Provincie-Zuid-Holland/Omgevingsbeleid-Frontend/blob/main/standard-for-public-code-assessment.md">Assessement of the frontend towards to Standard for Public Code</a>
+      <li><a href="https://github.com/Provincie-Zuid-Holland/Omgevingsbeleid-Frontend/blob/main/standard-for-public-code-assessment.md">Assessment of the frontend towards to Standard for Public Code</a>
```

Comment:
```suggestion
      <li><a href="https://github.com/Provincie-Zuid-Holland/Omgevingsbeleid-Frontend/blob/main/standard-for-public-code-assessment.md">Assessment of the frontend towards the Standard for Public Code</a>
```

Reply:
Even though I agree about the grammar, I find it a bit odd to make this change when merging back from main as it is mostly meant to sync the branches and this change would mean that they are not in sync after merge.

Reply:
Then let's make it in Develop.